dbt/adapters/redshift/connections.py

```diff
@@ -146,7 +177,17 @@ def get_connect_method(self):
             )
 
         if self.credentials.sslmode:
-            kwargs["sslmode"] = self.credentials.sslmode
+            ssl_args, warning_msg = _translate_sslmode(self.credentials.sslmode)
+            logger.warning((self.credentials.sslmode).lower())
+            if warning_msg is not None:
+                logger.warning(warning_msg)
+            kwargs["ssl"] = ssl_args["ssl"]
+            kwargs["sslmode"] = ssl_args["sslmode"]
+        elif self.credentials.sslmode is None:
+            logger.warning(
+                "Establishing connection using ssl with sslmode set to 'verify-ca'. "
+                "To connect without ssl, set sslmode to 'disable'."
+            )
```

Comment:
If "sslmode" is not detected in profiles.yml, it will go into this condition of `elif self.credentials.sslmode is None:`
if sslmode is provided as "None", it will go into the  condition of ` if self.credentials.sslmode`.
`elif self.credentials.sslmode is None:` and ` if self.credentials.sslmode` are mutually exclusive and therefore if we move` elif self.credentials.sslmode is None` within the ` if self.credentials.sslmode`, it will never be executed

Reply:
Now that I look at the `_translate_sslmode` function. It seems like we already have logic for [`self.credentials.sslmode` being `None`](https://github.com/dbt-labs/dbt-redshift/pull/439/files#diff-d9550a771df77b5d9cdf398167175d989046ecc22dd79cddb579b196216ddec0R131), can't we move this there?

Reply:
the line "elif sslmode_input == "none":" handles the case where profiles.yml is configured as sslmode = None